docs/content/topics/scripting/advanced.md

```diff
@@ -37,188 +47,207 @@ function exec(e, p) {
     .then(() => {
         console.log("done");
     });
-};
+}
+
+events.process();
 ```
-[advanced-01.js](../../examples/advanced-01.js)
 
-In the example above, we use implicit JavaScript `Promise` objects for chaining two jobs, then printing `done` after the two jobs are run. Each `Job.run()` call returns a `Promise`, and we call that `Promise`'s `then()` method.
+In the example above, we use implicit JavaScript `Promise` objects for chaining
```

Comment:
I'm questioning the "implicit" part of this statement. Runnables do literally return `Promise<void>`.